scripts/BooksAPI.php

```diff
@@ -177,13 +178,16 @@ function saveBooksToDatabase()
     $books = $this->getBooks();
     $bookModel = new BookModel();
     foreach ($books as $book) {
-      $book['authorsIds'] = $this->saveTerms($book['authors'], 'authors');
-      $book['categoriesIds'] = $this->saveTerms($book['categories'], 'categories');
-      unset($book['authors']);
-      unset($book['categories']);
-      $bookEntity = new BookEntity($book);
-      $bookModel->save($bookEntity);
+      if (!$bookModel->checkBook($book)) {
```

Comment:
it's simpler to remove negation and put continue right after if, instead of enclosing entire code into an if and add an else also.